Translated/Dialog/Dighal/ssn_011110.csv

```diff
@@ -0,0 +1,13 @@
+ssn_011110_0001#0,"""Have you tried your hand at Gathering?"""
+ssn_011110_0002#0,"""Outside the city, there's a wealth of vegetables, fruits, and Tames meat to be harvested."""
```

Comment:
Outside the city, there's a wealth of vegetables, fruits, Ores and Tames meat to be harvested.

Reply:
oops, missed that. good eye

Reply:
fixed, cheers